.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Run E2E Playwright tests
+on:
+  push:
```

Comment:
which actions should we allow/have here? I think it makes sense to allow manual runs and otherwise do it nightly?

Reply:
I copied from our [CLI test setup](https://github.com/github/vscode-codeql/blob/main/.github/workflows/cli-test.yml#L113) and decided to run them on push to main, merge to main and every night of the week.